### PR TITLE
implement exact matching classification

### LINKED ISSUE #53: kNN exact matching
Implement exact matching in the kNN classifier, and use smaller SDR dimensions n=100 and w=20.
See [`dist` in the classifier code](https://github.com/numenta/nupic/blob/master/nupic/algorithms/KNNClassifier.py#L593).

### LINKED ISSUE #54: Use numpy arrays
The kNN classifier expects patterns as numpy arrays; lists work fine, but there are errors when trying to run the classifier with `verbosity` > 0.

### PR DESCRIPTION
**Pull request by BoltzmannBrain**
- fixes #53
- specifies kNN verbosity
- fixes #54; use numpy arrays for data going to the classifier

(Not yet ready for review.)

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@subutai please review. Also, this is blocked on the [NuPIC PR for kNN exact matching classifications](https://github.com/numenta/nupic/pull/2169).

File: fluent/models/classify_randomSDR.py [hunk 1]
```diff
@@ -31,32 +32,39 @@ class ClassificationModelRandomSDR(ClassificationModel):
   Class to run the survey response classification task with random SDRs.
   """
 
-  def __init__(self, verbosity):
+  def __init__(self, 
+               exact=True,
```
Comment (subutai) [inline fluent/models/classify_randomSDR.py]: Why is this an option? When would we not want exact matching for random SDRs?

**Comment by subutai** [comment]
Just one comment. Otherwise :+1:

File: fluent/models/classify_randomSDR.py (hunk 1 above)
Comment (BoltzmannBrain) [inline fluent/models/classify_randomSDR.py]: Good call :smile:

**Comment by subutai** [comment]
:+1:

**Pull request merged**